drivers/wifi/esp/esp.c

```diff
@@ -720,6 +725,16 @@ static void esp_init_work(struct k_work *work)
 		SETUP_CMD_NOHANDLE("AT"),
 #endif
 		SETUP_CMD_NOHANDLE("AT+"_CWMODE"=1"),
+#if CONFIG_WIFI_ESP_IP_STATIC
+		/* enable Static IP Config */
+		SETUP_CMD_NOHANDLE(ESP_CMD_DHCP_ENABLE(1, 0)),
+		SETUP_CMD_NOHANDLE(ESP_CMD_SET_IP(CONFIG_WIFI_ESP_IP_ADDRESS,
+						  CONFIG_WIFI_ESP_IP_GATEWAY,
+						  CONFIG_WIFI_ESP_IP_MASK)),
+#else
+		/* enable DHCP */
+		SETUP_CMD_NOHANDLE(ESP_CMD_DHCP_ENABLE(1, 1)),
```

Comment:
Should we add `ESP_MODE_SOFTAP` and `ESP_MODE_STATION` macros, so above line will get more readable:
```
SETUP_CMD_NOHANDLE(ESP_CMD_DHCP_ENABLE(ESP_MODE_STATION, 1)),
```
?